nixos/modules/misc/ids.nix

```diff
@@ -414,6 +415,7 @@
       btsync = 113;
       #minecraft = 114; # unused
       #monetdb = 115; # unused (not packaged), removed 2016-09-19
+      vault = 115;
```

Comment:
adjusting of ownership is removed (it was just a migration hack as the service user was downpriveleged from root to vault)